extras/userdoc/generator/README.md

```diff
@@ -0,0 +1,24 @@
+# README for the new NEST Help Generator
+
+Processing more in less time with less code.
```

Comment:
This slogan does not explain anything to a normal user, who does not know the horrors of the old SLI parser. Please remove it.